TriangleChecker.java

```diff
@@ -48,19 +48,19 @@ private static void printAnalysis(float a, float b, float c) {
 
   // Analyse der Dreiecksart
   public static TriangleType checkTriangle(float a, float b, float c) {
+      if (((a > 0) && (b > 0) && (c > 0)) && ((a + b >= c) && (b + c >= a ) && (a + c >= b))) {        
+
       if ((a > 0) && (b > 0) && (c > 0)) {        
```

Comment:
Ist diese Prüfung notwendig, wnn das Gleiche in der Zeile vorher schon geprüft wird?